GitTfs/Core/DerivedGitTfsRemote.cs

```diff
@@ -190,6 +190,12 @@ public string Prefix
         public bool ExportMetadatas { get; set; }
         public Dictionary<string, string> ExportWorkitemsMapping { get; set; }
 
+        public int? InitialChangeset
```

Comment:
Hmm, I'm not intimately familiar with this code here--but is there something different we can do here? Why have a property that says it get/sets an `int?`, but in reality, all it does is throw an exception? That's a lying contract that's ripe for producing errors.

Until I look at the code in more detail, I'm going to make some conjectures. Based on the name of this file, I'm making as assumption that this class inherits from some base class. Perhaps the base class should be changed to better accommodate the behavior you need from derived classes? OR, perhaps the base class should implement some interface and provide a default, `virtual` implementation (or, if the base class is abstract, make this an abstract property). OR perhaps this should really be a method on the base class that has an explicit contract that allows for this exception to be thrown in certain circumstances?